features/api/v1/licenses/index.feature

```diff
@@ -2151,7 +2151,7 @@ Feature: List license
     And the current user has 1 "license" as "owner"
     When I send a GET request to "/accounts/test1/licenses?user=ef8e7a71-6b54-4a9b-8717-778516c9ad25%27%20or%201=1"
     Then the response status should be "200"
-    And the response body should be an array with 0 "licenses"
+    And the response body should be an array with 1 "license"
```

Comment:
This does NOT signal a SQL injection — this is returning the current user only, since the authz scoping overwrite the provided query parameter scoping. Previously, the scopes were combined, but since you can't combine `.from(...)` calls, the latter `.from(...)` call from `authorized_scope` overwrites the prior from `apply_scopes`.